cypress/support/commands.js

```diff
@@ -32,10 +32,56 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
+Cypress.Commands.add('register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users', {
+      email,
+      username,
+      password
+    });
```

Comment:
Ensure that the '/users' endpoint is correct and accessible for the `register` command to function properly. This endpoint should handle user registration.